exercises/simple-linked-list/src/test/scala/SimpleLinkedListTest.scala

```diff
@@ -0,0 +1,56 @@
+import org.scalatest.{FlatSpec, Matchers}
+
+class SimpleLinkedListTest extends FlatSpec with Matchers {
+
+  it should "handle single item list" in {
+    val list = SimpleLinkedList().add(1)
+    list.isEmpty should be (false)
+    list.value should be (1)
+  }
+
+  it should "handle single item has no next item" in {
```

Comment:
Just as a remark:  
This would also be a perfect case for [Property-based testing](http://www.scalatest.org/user_guide/property_based_testing).

But it might be too demanding for most people I guess. And it is not used anywhere.